test/feeds.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require('supertest');
+
+const app = require('../src/backend/web/app');
+const Feed = require('../src/backend/data/feed');
+const { addFeed } = require('../src/backend/utils/storage');
+const hash = require('../src/backend/data/hash');
+
+describe('test /feeds endpoint', () => {
+  const createdItems = 150;
+
+  // Array of feeds
+  const feeds = [...Array(createdItems).keys()].map(item => {
+    const url = `http://telescope${item}.cdot.systems`;
+    return {
+      id: hash(url),
```

Comment:
Instead of doing the id and hashing manually, let's use `new Feed()` and then `await feed.save()` below in your `beforeAll` and have it do it for us automatically.  That will also help to test that code more thoroughly.  Ideally the details of how the hashes and the like work will be opaque to this code.